xml/FinanceType.xml

```diff
@@ -9,7 +9,19 @@
         <url>http://www.oecd.org/dac/stats/dacandcrscodelists.htm</url>
     </metadata>
     <codelist-items>
-        <codelist-item>
+        <codelist-item status="active">
+            <code>1</code>
+            <name>
+                <narrative>GNI: Gross National Income</narrative>
+                <narrative xml:lang="fr">RNB : Revenu national brut</narrative>
+            </name>
+            <description>
+                <narrative/>
+                <narrative xml:lang="fr"/>
+            </description>
+            <category>0</category>
```

Comment:
I guess any FinanceType codes with category 0 probably shouldn’t be added, for the same reason as https://github.com/IATI/IATI-Codelists-NonEmbedded/pull/283/files#r260298899

Reply:
Marking as resolved, as per Petya’s comment here: https://github.com/IATI/IATI-Codelists-NonEmbedded/pull/283#discussion_r